tests/extended/test_trainer_ext.py

```diff
@@ -71,6 +73,17 @@ def require_apex(test_case):
         return test_case
 
 
+# a candidate for testing_utils
+def require_bnb(test_case):
```

Comment:
Should be named `require_bitsandbytes` (like above) and go in `testing_utils`.

Reply:
Sure, so should I move these 2 as well to `testing_utils.py`?

https://github.com/huggingface/transformers/blob/78f346c2b5164695ff4aecc27e2438545f14f9fa/tests/extended/test_trainer_ext.py#L52-L71

or in another PR?

we kept those local since they weren't yet used anywhere else besides this test.

Reply:
This PR or another one, as you prefer. They should all be centralized if possible.

Reply:
OK, all done in now in this PR.

@sgugger, please kindly have one last look before we merge this